gpytorch/kernels/linear_kernel.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch import nn
+from .kernel import Kernel
+
+
+class LinearKernel(Kernel):
+    def __init__(
+        self,
+        num_dimensions,
+        offset_bounds=(-10000, 10000),
+        linear_coefficient_bounds=(-10000, 10000),
+        eps=1e-5,
+        active_dims=None,
+    ):
+        super(LinearKernel, self).__init__(active_dims=active_dims)
+        self.eps = eps
+        self.register_parameter(
+            'offset',
+            nn.Parameter(torch.zeros(1, 1)),
+            bounds=offset_bounds,
+        )
+        self.register_paramter(
+            'coefficients',
+            nn.Parameter(torch.zeros(1, num_dimensions)),
+            bounds=linear_coefficient_bounds,
+        )
```

Comment:
This will probably change once we add the ARD option (since "lengthscale" will probably become a parameter on the Kernel superclass), but it's fine for now!

Also, I think these parameters should probably be 3D tensors rather than matrices (i.e. `torch.zeros(1, 1, num_dimensions)`. The base Kernel class is designed to operate on batches of data (batch_size x n_data x n_dims).